main.js

```diff
@@ -364,6 +364,16 @@ function windowCreate(json) {
         sessionCreate(elements[json.targetID].webContents, json.sessionId)
         client.write(json.targetID, consts.eventNames.windowEventDidFinishLoad)
     })
+    elements[json.targetID].webContents.on('did-get-redirect-request', (event, oldUrl, newUrl) => {
```

Comment:
I'd rather you send both `oldUrl` and `newUrl` in the payload.
That means you'll have to add them as well in the [GO event](https://github.com/asticode/go-astilectron/blob/master/event.go#L32)